querydsl-jdo/src/test/java/com/querydsl/jdo/BasicsTest.java

```diff
@@ -93,10 +94,14 @@ public void list_distinct() {
 
     @Test
     public void list_distinct_two_sources() {
-        // XXX List implementation of JDO provider has weird equals implementation
-        assertEquals(
-                ImmutableList.copyOf(query().from(product, product2).select(product, product2).fetch()),
-                ImmutableList.copyOf(query().from(product, product2).distinct().select(product, product2).fetch()));
+        try {
+            // XXX List implementation of JDO provider has weird equals implementation
```

Comment:
It's a bug in the underlying JDO implementation, and we cover lots of Hibernate bugs, so users might also expect that we cover this one.